test/journeys/lib/test-users.js

```diff
@@ -179,3 +179,113 @@ export function disconnectDevices(users) {
     user.spark.internal.mercury.disconnect());
   return browser.call(() => Promise.all(promises));
 }
+
+/**
+ * Unmute message notification
+ * @param {Object} spark
+ * @param {String} convoId
+ * @returns {Object}
+ */
+export function unmuteMessageNotification(spark, convoId) {
```

Comment:
Awesome helpers, but can we move them out of the `test-users` helper library? Maybe a `convo.js` file?